esp/templates/program/modules/bigboardmodule/bigboard.html

```diff
@@ -50,12 +52,29 @@
             type: 'datetime',
             minRange: 24 * 60 * 60 * 1000, // one day
           },
-          yAxis: {
+          yAxis: [{
             title: {text: null},
             min: 0,
             maxPadding: 0.01,
             endOnTick: false,
+          }
+          {% if hours_data %}
+          ,{
+            title: {text: 'Class-Hours'},
```

Comment:
Can these at least use `{{hours_data.0.description}}` or something like below?  It's a little weird to hardcode them here as the names of the second axis.  (It may also be better to call it `left_axis_data` and `right_axis_data` rather than `graph_data` and `hours_data`.)